Scripts/Insights_19.10.3_hotfix.ps1

```diff
@@ -0,0 +1,60 @@
+﻿param(
+   [Parameter(Mandatory=$true)][string]$username
+  ,[Parameter(Mandatory=$true)][string]$password
+)
+
+$insightsAdminToolPath = "${Env:ProgramFiles(x86)}\UiPath\Orchestrator\Tools"
+$uifrostPath = "${Env:ProgramFiles}\Sisense\DataConnectors\JVMContainer\Connectors\UiFrost"
+
+function Get-CheckBuild($tenant)
+{
+	$cubeBuilding = 1
+	while ($cubeBuilding -ne 0)
+	{
+		$buildResult = & "$insightsAdminToolPath\UiPath.InsightsAdminTool.exe" buildStatus -t $tenant | out-string
+		$buildStatus = $buildResult.Split(“`n”)
+
+		if($buildStatus[1].StartsWith("Build successfully ended"))
+		{
+			$cubeBuilding = 0;
+		}
+		elseif ($cubeBuilding -eq 180)
+		{
+			throw 'Cube is still building after 30m. Please execute script during downtime.'
+		}
+		else
+		{
+			Write-Output "$tenant Cube is still building. Waiting..."
+			Start-Sleep -s 10
+			$cubeBuilding++
+		}
+	}
+}
+
+Write-Output "Copying .jar"
+Copy-Item com.sisense.connectors.jdbc.UiFrost.jar -Destination "$uifrostPath\com.sisense.connectors.jdbc.UiFrost.jar" -Recurse -force
```

Comment:
Can Copy-Item fail or throw if file cannot be overriden? If so we should treat that exception.